crates/compiler/parse/src/expr.rs

```diff
@@ -226,8 +226,7 @@ fn loc_term<'a>(options: ExprParseOptions) -> impl Parser<'a, Loc<Expr<'a>>, EEx
 }
 
 fn ident_seq<'a>() -> impl Parser<'a, Loc<Expr<'a>>, EExpr<'a>> {
-    (|arena: &'a Bump, state: State<'a>, min_indent: u32| parse_ident_seq(arena, state, min_indent))
-        .trace("ident_seq")
+    parse_ident_seq.trace("ident_seq")
```

Comment:
a clippy error reported by later versions. This replaces the closure by the function itself, a trivial change.